extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/recording/RecordableBootstrap.java

```diff
@@ -376,23 +376,6 @@ private void applyServiceContributors() {
         return providedServices;
     }
 
-    /**
-     * Temporarily exposed since Configuration is still around and much code still
-     * uses Configuration. This allows code to configure the builder and access that
-     * to configure Configuration object (used from HEM atm).
-     *
-     * @return The settings map.
-     *
-     * @deprecated Temporarily exposed since Configuration is still around and much
-     *             code still uses Configuration. This allows code to configure the
-     *             builder and access that to configure Configuration object.
-     */
-    @Override
-    @Deprecated
-    public Map getSettings() {
-        return settings;
```

Comment:
I'm not sure at all this one should be removed. @Sanne ?

Reply:
Don't remove this please.   The method is deprecated because it overrides a method from Hibernate ORM upstream which is deprecated, but it's still used and we need the override.

Also no big point in removing this, this is internal stuff.

Reply:
That's what I thought. I would say let's keep the method but drop the `@Deprecated` from it. If it gets removed from ORM at some point, we have the `@Override` so we will have a compilation failure until we drop it. That way we won't remove it next time.

Reply:
I'll leave it as `@Deprecated` for now to discourage people from using it (also because the overridden method is also deprecated). Hopefully when the deprecated method in Hibernate is removed this should be removed as well